redis_schedule_job.go

```diff
@@ -232,19 +232,19 @@ func (t *scheduleJob) doConsumeZset(ctx context.Context, vals []string, consumer
 func (t *scheduleJob) sendToStream(ctx context.Context, msg *Message) error {
 	mmsg := messageToMap(msg)
 	subType := normalSubscribe
-	if msg.MoodType == "sequential" {
+	if msg.MoodType == string(SEQUENTIAL) {
```

Comment:
no need `string(...)`

Reply:
@kongliang1 did you fix this one?

Reply:
There is no need to make any modifications here.  because data type has changed.